From ed30ef9013e41eed7a85c79a2ae066fcc13a1d9f Mon Sep 17 00:00:00 2001
From: Paolo Bonzini <pbonzini@redhat.com>
Date: Tue, 22 Jul 2014 14:38:20 -0400
Subject: [virt] kvm/vmx: error message typo fix

Message-id: <1406040016-3289-106-git-send-email-pbonzini@redhat.com>
Patchwork-id: 86184
O-Subject: [RHEL7 PATCH v2 105/221] kvm/vmx: error message typo fix
Bugzilla: 1116936
RH-Acked-by: Radim Krcmar <rkrcmar@redhat.com>
RH-Acked-by: Bandan Das <bsd@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: "Michael S. Tsirkin" <mst@redhat.com>

mst can't be blamed for lack of switch entries: the
issue is with msrs actually.

Signed-off-by: Michael S. Tsirkin <mst@redhat.com>
Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>
(cherry picked from commit 602662047594e62927202e38dde37d7fbcca4544)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/x86/kvm/vmx.c b/arch/x86/kvm/vmx.c
index ff46c6b..9e0cd6f 100644
--- a/arch/x86/kvm/vmx.c
+++ b/arch/x86/kvm/vmx.c
@@ -1498,7 +1498,7 @@ static void add_atomic_switch_msr(struct vcpu_vmx *vmx, unsigned msr,
    break;
 
  if (i == NR_AUTOLOAD_MSRS) {
-  printk_once(KERN_WARNING"Not enough mst switch entries. "
+  printk_once(KERN_WARNING "Not enough msr switch entries. "
     "Can't add msr %x\n", msr);
   return;
  } else if (i == m->nr) {
-- 
1.7.1